src/Workspaces/CSharp/Portable/Extensions/ExpressionSyntaxExtensions.cs

```diff
@@ -45,25 +45,27 @@ public static ExpressionSyntax WalkDownParentheses(this ExpressionSyntax express
             return expression;
         }
 
-        public static ExpressionSyntax Parenthesize(this ExpressionSyntax expression, bool includeElasticTrivia = true)
+        public static ExpressionSyntax Parenthesize(
```

Comment:
:memo: This was an unnecessarily complex way to add the `addSimplifierAnnotation` parameter, but now that I've reviewed it, it might as well stay. Do keep in mind that every time something like this happens, things just take more time.

Reply:
Sorry, could you clarify?  I'm not seeing a simpler way to do things?

Reply:
I think this comes down to a personal preference take.  I find your code to be less clear.  But i def can appreciate that different patterns and styles can impact the review.  I'll try to find an appropriate balance that allows for expressiveness and cleanliness, without overdoing it on the PR side.